scripts/cangen/pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "cangen"
+version = "0.1.0"
+description = "Converts .dbc files into C++ files for a CAN bus system."
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
```

Comment:
where did you come up with these dependencies? I feel like most of them are sub-dependencies of Jinja2 and cantools. We should only list the dependencies that we directly use and let `pip` figure out the sub-dependencies.

To simplify this list, remove all dependencies (besides `python`) and `pip install` this project into a fresh virtual environment. Then run `cangen` and see which packages are missing. Start re-adding the packages, starting with `Jinja2` and `cantools`, until you don't have any errors.

You should use `poetry add package_name` to add the packages instead of manually writing them into the toml file. See <https://python-poetry.org/docs/cli/#add>